_docs/policies/codeowners.md

```diff
@@ -0,0 +1,28 @@
+# Code Owners Policy
+
+### Purpose
+
+This policy outlines how code ownership is assigned and ensures that core contributors (CCs) are accountable for 
+reviewing and maintaining code in the `status-go` repository.
+
+### Code Owners File
+
+- Code owners MUST be stored in the `.github/CODEOWNERS` file, as per GitHub's [CODEOWNERS documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+
+### Assigning Code Ownership
+
+1. Eligibility
+ 
+- Only CCs MAY be code owners. External contributors are not eligible to be code owners.
+
+2. Nomination
+ 
+- For **existing** code:
+  - Any CC MAY nominate themselves as a code owner for any part of the existing code.
+  - Nominations SHOULD be discussed and agreed upon with other CCs if there are overlaps or concerns.
+- For **new** code:
+  - When introducing a new file/module, CCs SHOULD assign themselves or another CC as a code owner for this file/module. 
```

Comment:
But do we need an owner for every new **file** created from now on? Seems excessive to me to manage this at the file level. On the other hand, if a new **module** is being introduced, then I agree with the nomination rule above.

Reply:
I agree that this might become an overkill, but for some huge modules (e.g. `protocol`) I believe there should be such an option (at least for now). Unfortunately, we can easily add a new part of `Messenger` in a separate file, and it would make sense to add a code owner, while it wouldn't make sense for the whole `protocol` module.

I have no strong opinion on this

Reply:
What about something like

```
When introducing a new module or substantial new component, CCs SHOULD assign ownership. File-level ownership MAY be used where appropriate but is not required.
```

I think per file level changes may be overly cumbersome.

Reply:
@Samyoul's suggestion above strike a good balance to me.

Reply:
@ilmotta @Samyoul thank you guys, I've added Sam's suggested text in f1807194a04e5c7641d6bf29c1e7d97e91e9828a.